src/plugins/janus_streaming.c

```diff
@@ -2110,6 +2132,11 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 					cl = cl->next;
 					continue;
 				}
+				if(abscaptime_src_id && abscaptime_src_id->value && (atoi(abscaptime_src_id->value) < 1 || atoi(abscaptime_src_id->value) > 14)) {
```

Comment:
Why not assign the value of the int to a variable just once, and then check/use the variable? The way you're doing it now, you're performing an `atoi` multiple times: twice here, once more when invoking the mountpoint creation.

Reply:
I've tried to do this, but imo it adds more complexity due to additional if statements as we should differentiate the cases when abscaptime_src_id property has been set to 0 by user or by code (as default int value).

Reply:
Not sure I understand what the problem would be: even if the configuration property sets a value of 0, that's equivalent to not have the value at all, so the default. I'm mostly concerned about limiting the number of unneeded atoi calls, that are done multiple times for the same property when once would suffice.

Reply:
Ok, if we are okay with 0 value in configuration I will change it in next commit

Reply:
@lminiero fixed